olive/systems/docker/utils.py

```diff
@@ -88,36 +87,14 @@ def create_metric_volumes_list(
     return mount_list
 
 
-def create_resource_path_mount(
-    resource_path: ResourcePath, local_resource_path: ResourcePath, container_root_path: Path
-) -> (str, str):
-    # no need to mount if resource path is None or string name
-    if not resource_path or resource_path.is_string_name():
```

Comment:
This case is still needed. There is nothing to mount for None or string resources

Reply:
will add it.